notifications.bs

```diff
@@ -728,10 +800,12 @@ partial interface ServiceWorkerRegistration {
  Exposed=ServiceWorker]
 interface NotificationEvent : ExtendableEvent {
   readonly attribute Notification notification;
+  readonly attribute DOMString? action;
 };
 
 dictionary NotificationEventInit : ExtendableEventInit {
   required Notification notification;
+  DOMString? action = null;
```

Comment:
I would prefer special casing the empty string. That would match `tag` better iirc.